test/go/scenarios.json

```diff
@@ -35,8 +35,8 @@
             }
         }
     },
-    "install_go_centos-7": {
-        "image": "centos:centos7",
```

Comment:
Can we keep this as is? For the test failures, you can pull from `main` which would have a fix for it.